src/main/java/rx/observers/SafeCompletableSubscriber.java

```diff
@@ -52,8 +52,8 @@ public void onCompleted() {
 
     @Override
     public void onError(Throwable e) {
+        RxJavaPluginUtils.handleException(e);
```

Comment:
But why `SafeSubscriber` doesn't do this? Either this class or the other one are not behaving right.